backend/src/main/java/com/bakdata/conquery/models/preproc/parser/specific/StringParser.java

```diff
@@ -132,12 +131,18 @@ protected StringStore decideType() {
 		);
 
 		StringStore result = guess.getType();
+
+		if(result instanceof StringTypeEncoded){
+
+		}
+
```

Comment:
activates super mode

Reply:
use the force luke!

Reply:
wenn man mit dem Feuerzeug an den Bildschirm geht sieht man die geheimnachrichtt